# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/forms/nsFieldSetFrame.h
# Commit: 6141de941e71
# Full Hash: 6141de941e717cdb5485e1aa1ca00ffcd74f8a81
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 19:41:15
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 3cf752580f1b -r 6141de941e71 layout/forms/nsFieldSetFrame.h
--- a/layout/forms/nsFieldSetFrame.h	Fri Jan 29 19:20:24 2021 +0000
+++ b/layout/forms/nsFieldSetFrame.h	Sat Jan 30 13:47:10 2021 +0000
@@ -49,11 +49,11 @@
                             gfxContext& aRenderingContext, nsPoint aPt,
                             const nsRect& aDirtyRect);
 
-#ifdef DEBUG
   virtual void SetInitialChildList(ChildListID aListID,
                                    nsFrameList& aChildList) override;
   virtual void AppendFrames(ChildListID aListID,
                             nsFrameList& aFrameList) override;
+#ifdef DEBUG
   virtual void InsertFrames(ChildListID aListID, nsIFrame* aPrevFrame,
                             const nsLineList::iterator* aPrevFrameLine,
                             nsFrameList& aFrameList) override;
@@ -92,9 +92,8 @@
   nsContainerFrame* GetInner() const;
 
   /**
-   * Return the frame that represents the legend if any.  This may be
-   * a nsLegendFrame or a nsHTMLScrollFrame with the nsLegendFrame as the
-   * scrolled frame (aka content insertion frame).
+   * Return the frame that represents the rendered legend if any.
+   * https://html.spec.whatwg.org/multipage/rendering.html#rendered-legend
    */
   nsIFrame* GetLegend() const;
 